tests/utils/test_fmrisim.py

```diff
@@ -74,39 +75,40 @@ def test_generate_stimfunction():
     stimfunction = sim.generate_stimfunction(onsets=onsets,
                                              event_durations=event_durations,
                                              total_time=duration,
-                                             tr_duration=tr_duration,
                                              )
 
-    assert len(stimfunction) == duration / tr_duration, "stimfunction incorrect " \
+    assert len(stimfunction) == duration * 1000, "stimfunction incorrect " \
                                                         "length"
-    assert np.sum(stimfunction) == np.sum(event_durations * len(onsets)) / \
-                                   tr_duration, "Event number"
+    assert np.sum(stimfunction) == np.sum(event_durations * len(onsets)) * \
+                                   1000, "Event number"
```

Comment:
You should break the line before binary operators:
https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator